sections/design.tex

```diff
@@ -26,4 +26,6 @@ \section*{Design considerations}
 Some attendees, usually those with a strong background in hackathons and their specific topic of study, arrive with the goal of writing a scientific article.
 Other attendees plan to learn a specific topic, such as machine learning, or to analyze a specific data set that relates to the tools covered in the tutorials.
 This leads to a wide variety of project types from sandbox-style explorations to focused work efforts.
-This breadth of possible outcomes makes it difficult to design for all possible participant goals, and calls for adaptive, flexible leadership among hack week organizers.
\ No newline at end of file
+This breadth of possible outcomes makes it difficult to design for all possible participant goals, and calls for adaptive, flexible leadership among hack week organizers.
+The large variety in participant backgrounds and experiences can to inc that many participants will experience impostor syndrome (see also supplementary materials), and it 
```

Comment:
Oops, I should try to write a paper while at a conference. 😳 
Should be fixed now.